src/test/stepFunctions/commands/getStateMachineDefinitionFromCfnTemplate.test.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as getCfnDefinition from '../../../stepFunctions/commands/visualizeStateMachine/getStateMachineDefinitionFromCfnTemplate'
+
+const unescapedJsonString: string = `{"StartAt":"Submit Job","States":{"Submit Job":{"Next":"Wait X Seconds","Type":"Task","Resource":"","ResultPath":"$.guid"},"Wait X Seconds":{"Type":"Wait","SecondsPath":"$.wait_time","Next":"Get Job Status"},"Get Job Status":{"Next":"Job Complete?","InputPath":"$.guid","Type":"Task","Resource":"","ResultPath":"$.status"},"Job Complete?":{"Type":"Choice","Choices":[{"Variable":"$.status","StringEquals":"FAILED","Next":"Job Failed"},{"Variable":"$.status","StringEquals":"SUCCEEDED","Next":"Get Final Job Status"}],"Default":"Wait X Seconds"},"Job Failed":{"Type":"Fail","Error":"DescribeJob returned FAILED","Cause":"AWS Batch Job Failed"},"Get Final Job Status":{"End":true,"InputPath":"$.guid","Type":"Task","Resource":""}},"TimeoutSeconds":30}`
+const unescapedJsonString2: string = `{"StartAt":"Submit Job2","States":{"Submit Job2":{"Next":"Wait X Seconds","Type":"Task","Resource":"","ResultPath":"$.guid"},"Wait X Seconds":{"Type":"Wait","SecondsPath":"$.wait_time","Next":"Get Job Status"},"Get Job Status":{"Next":"Job Complete?","InputPath":"$.guid","Type":"Task","Resource":"","ResultPath":"$.status"},"Job Complete?":{"Type":"Choice","Choices":[{"Variable":"$.status","StringEquals":"FAILED","Next":"Job Failed"},{"Variable":"$.status","StringEquals":"SUCCEEDED","Next":"Get Final Job Status"}],"Default":"Wait X Seconds"},"Job Failed":{"Type":"Fail","Error":"DescribeJob returned FAILED","Cause":"AWS Batch Job Failed"},"Get Final Job Status":{"End":true,"InputPath":"$.guid","Type":"Task","Resource":""}},"TimeoutSeconds":30}`
+const uniqueIdendifier = 'MyStateMachine'
+const cdkOutPath = __dirname.replace('/dist', '') + '/resources'
+const templatePath = cdkOutPath + `/templateJsonTester.template.json`
+
+describe('Get State Machine Definition from Cfn Template', function () {
+    it('get the correct cfn definition for state machine with correct inputs', async function () {
+        let data = getCfnDefinition.getStateMachineDefinitionFromCfnTemplate(uniqueIdendifier, templatePath)
+        data = getCfnDefinition.toUnescapedAslJsonString(data as string)
```

Comment:
Why does this need to be specified as a string? The return type for `toUnescapedAslJsonString` is only string. Goes for elsewhere in this file.